preprocessing/nextclade/src/loculus_preprocessing/prepro.py

```diff
@@ -368,9 +483,12 @@ def enrich_with_nextclade(  # noqa: C901, PLR0914, PLR0915
         AccessionVersion, defaultdict[GeneName, list[AminoAcidInsertion]]
     ] = defaultdict(lambda: defaultdict(list))
     with TemporaryDirectory(delete=not config.keep_tmp_dir) as result_dir:  # noqa: PLR1702
-        for segment in config.nucleotideSequences:
-            result_dir_seg = result_dir if segment == "main" else result_dir + "/" + segment
-            dataset_dir_seg = dataset_dir if segment == "main" else dataset_dir + "/" + segment
+        for sequence_and_dataset in config.nucleotideSequences:
```

Comment:
weird error:
```
Error:
   0: --input-dataset: path is invalid. Expected a directory path, a zip file path or json file path, but got: '"/tmp/tmpxutlxf0a/CV-A16"'
Location:
   packages/nextclade-cli/src/dataset/dataset_download.rs:45
```

Reply:
I have no clue why this error is now turning up but isn't on the previews - I haven't changed this part of the code... it seems like nothing is working today

Reply:
even with the strip code I get the same error:
```
Error:
   0: --input-dataset: path is invalid. Expected a directory path, a zip file path or json file path, but got: '"/tmp/tmp0wm8mky3/L"'
```

this doesn't make any sense - why isn't this seen in the test?????

Reply:
Ah but maybe the temp dir isn't being made correctly! This is sth the tests don;t check!

Reply:
Yeah exactly – not that Nextclade is the source of the bug just that these quotes aren't it